source/features/one-click-pr-or-gist.tsx

```diff
@@ -1,11 +1,12 @@
+import './one-click-pr-or-gist.css';
 import React from 'dom-chef';
 import select from 'select-dom';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
 function init(): void | false {
-	const draftPROption = select('.new-pr-form [name="draft"]');
+	const draftPROption = select('.new-pr-form [name="draft"], #new_gist [name="gist[public]"]');
 	if (!draftPROption) {
 		// 1. Free accounts can't open Draft PRs in private repos, so this element is missing
```

Comment:
Is the above line still true?

Reply:
Yes, but now that I tested it in a draft PR page (I thought it was ok, based on your [comment](https://github.com/sindresorhus/refined-github/issues/4366#issuecomment-846079038)), 
I see now I have to convert
https://github.com/darkred/refined-github/blob/16bd5c23e27b4ac837d6de30bea821a081ffeeb1/source/features/one-click-pr-or-gist.tsx#L24-L28 

to 

```js
if (/\bdraft\b/i.test(title)) {
	title = 'Create draft PR';
}
classList.push('btn-primary');
```

otherwise in PR page, only the button to the right becomes green:
![2021-05-23_174453](https://user-images.githubusercontent.com/723651/119265270-bf3d2c80-bbee-11eb-8299-dad00d7fd1d2.jpg)

&nbsp;
Am I missing something, or should I make the change?

Reply:
Its fine I don't think it needs to be a different color

Reply:
Ok, thank you.

Reply:
Checked, yes